tap_github/streams.py

```diff
@@ -938,10 +939,95 @@ class ContributorsStream(GitHubStream):
         th.Property("repos_url", th.StringType),
         th.Property("events_url", th.StringType),
         th.Property("received_events_url", th.StringType),
-        # Anonymous contributor keys. Only fetched with params["anon"] = 'true'
+        th.Property("type", th.StringType),
+        th.Property("site_admin", th.BooleanType),
+        th.Property("contributions", th.IntegerType),
+    ).to_dict()
+
+
+class AnonymousContributorsStream(GitHubStream):
```

Comment:
Could you define this as a child stream of `ContributorsStream`? Then you run the first stream, pass the number of non-anonymous contributors you received as some sort of context, and then run the child stream and start paginating from where the parent stream left off?
It's probably abusing the parent-child relationship a bit, and I don't know if there might be weird side effects as a result.

Reply:
Anonymous contributors are not necessarily at the end either... so I think that would be tricky.

Reply:
arf, ok. Thanks github. Maybe there's a way to produce 2 different types of record messages for a single stream? But I don't know how :)